hazelcast/src/main/java/com/hazelcast/query/extractor/ValueReader.java

```diff
@@ -29,29 +29,29 @@
  * <p>
  * The wildcard quantifier may be used a couple of times, like person.leg[any].finger[any] which returns all fingers
  * from all legs.
- *
- * @param <T> Type of the value to read
  */
-public abstract class ValueReader<T> {
+public abstract class ValueReader {
 
```

Comment:
Didn't make sense to have it here, since a reader may be used to read various paths returning various types.